src/style/star.css

```
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars .stars__star:nth-child(n+5) {
+  background-image: url(../images/star.svg);
+}
```

```suggestion
.stars__star:nth-child(n+5) {
  background-image: url(../images/star.svg);
}
```цього буде достатньо